README.md

```diff
@@ -7,19 +7,252 @@ inspectpack
+
+```yaml
+- my-app:             # Resolved
+  - lodash@^4.1.0     # 4.2.3
+  - one@1.2.3:        # 1.2.3
+    - lodash@^3.0.0   # 3.1.0
+  - two@2.3.4:        # 2.3.4
+    - lodash@^3.0.0   # 3.1.0
+```
+
+Using modern `npm` or `yarn` to install this project gives us the following on-disk `node_modules` folder (with version resolutions noted):
+
+```bash
+node_modules          # Resolved
+  lodash              # 4.2.3
+  one                 # 1.2.3
+    node_modules
+      lodash          # 3.1.0 (Cannot be collapsed)
+  two                 # 2.3.4
+    node_modules
+      lodash          # 3.1.0 (Cannot be collapsed)
+```
+
+Looking to our resulting [bundle](https://github.com/FormidableLabs/inspectpack-duplicates-blog-examples/blob/master/scenarios/new-webpack/new-npm-unflattened/dist/bundle.js) we have the following duplicated code sources:
+
+- `node_modules/lodash/index.js` (`4.2.3`): This code is **similar** to the code for `3.1.0`.
+- `node_modules/one/node_modules/lodash/index.js`, `node_modules/two/node_modules/lodash/index.js`  (`3.1.0`): These two files are **identical** code sources. They are only included twice in the bundle because `npm`/`yarn` could not flatten the dependencies during installation.
+
+So, we've got inefficient code that we discovered via a _manual_ inspection. Wouldn't it be nice to have a report that specifically highlighted problems like these with useful information?
+
+... enter the `DuplicatesPlugin`.
+
+#### Diagnosing duplicates
+
+##### Simple report
+
+With our plugin enabled in the standard configuration:
+
+```js
+new DuplicatesPlugin()
+```
+
+we get a summary report of the duplicates upon running the `webpack` command:
+
+```
+WARNING in Duplicate Sources / Packages - Duplicates found! ⚠️
+
+* Duplicates: Found a total of 2 similar files across 3 code sources (both identical + similiar) accounting for 703 bundled bytes.
+* Packages: Found a total of 1 packages with 2 resolved, 3 installed, and 3 depended versions.
+
+## bundle.js
+lodash (Found 2 resolved, 3 installed. Latest version 4.2.3.)
+  3.1.0 ~/one/~/lodash
+    scenario-new-webpack-new-npm-unflattened@* -> one@1.2.3 -> lodash@^3.0.0
+  3.1.0 ~/two/~/lodash
+    scenario-new-webpack-new-npm-unflattened@* -> two@2.3.4 -> lodash@^3.0.0
+  4.2.3 ~/lodash
+    scenario-new-webpack-new-npm-unflattened@* -> lodash@^4.1.0
+```
+
+Breaking down this report, we get a `webpack` "warning" emitted by default with an initial summary
+of the report.
+
+* The `Duplicates` summary looks at **what is in the `webpack` bundle**. It tells us there are 2 files that are not identical, but the same package file path (e.g `3.1.0` vs `4.2.3` for `lodash/index.js`) and that there are 3 code sources that end up in our final bundle (which includes _two_ for `4.2.3`). We also get a byte count for all the files at issue (`703` bytes), which presumably could roughly be cut by 2/3 if we could collapse to just **one** file to do the same thing.
```

Comment:
This section indicates that there are two code sources for `lodash@4.2.3`, but I think the scenario is that there are two for `lodash@3.1.0`, correct.

> which includes _two_ for `4.2.3`.

Reply:
Yup!